doc/source/api-reference/qibo.rst

```diff
@@ -129,6 +129,11 @@ and the second will act to ``(1, 2)`` corresponding to
 
     [Y(1), Z(2), CNOT(1, 2), H(1), H(2)]
 
+Greenberger-Horne-Zeilinger (GHZ) state
+"""""""""""""""""""""""""""""""""""""""
+
+.. autofunction:: qibo.models.encodings.ghz_state
+
```

Comment:
For consistency, this should be in the `models.encodings` section of the API reference.

Reply:
I initially added `ghz_state` to the the Circuits section as the function also generates a circuit. Yup agree with consistency, so I've shifted to the Encoders section.